app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from os import path, getcwd
+
+
+def copy_file(command: str) -> str or None:
+
+    try:
+        cmd, main_file, new_file = command.split()
+        source = fr"{getcwd()}\{main_file}"
+
```

Comment:
consider removing this line due to the fact, that we can work with relative paths here. theres no need to get our working dir at all, tbh, but still, it is better to use `getcwd` than `sys.path[0]` if you insist. Also, If you wish to keep you `source` path as an absolute one, you should use `os.path.join` to make it